src/bin/common/parse_command.ml

```diff
@@ -1230,33 +1210,45 @@ let parse_theory_opt =
 let parse_fmt_opt =
 
   let docs = s_fmt in
+  let docv = "CHANNEL" in
 
-  let std_formatter =
-    let doc = Format.sprintf
-        "Set the standard formatter used by default to output the results,
+  let std_output, deprecated_std_output =
+    let doc =
+      Format.sprintf
+        "Set the standard output used by default to print the results,
     models and unsat cores. Possible values are %s."
-        (Arg.doc_alts ["stdout"; "stderr"; "<filename>"]) in
-    Arg.(value & opt formatter_conv Stdout & info ["std-formatter"] ~docs ~doc)
+        (Arg.doc_alts ["stdout"; "stderr"; "<filename>"])
+    in
+    let deprecated = "this option is depreciated. Please use --std-output." in
```

Comment:
Does `cmdliner` displays what option this refers to, or should this be `--std-formatter is deprecated, please use --std-output instead`? The information about deprecation should also be present in `--help`, I am not familiar with `~deprecated` so not sure if it is.

Reply:
Yes, cmdliner displays the option in the warning message and there is a deprecated tag in the man page also.